entity-framework/core/get-started/general/index.md

```diff
@@ -10,6 +10,6 @@ ms.technology: entity-framework-core
 
 uid: core/get-started/general/index
 ---
-# General Installation Instructions
+# General Installation Instructions for EF Core
```

Comment:
This section is also new. I added it as a substitute of having this repeated every time we post to the blog :relieved: